### ISSUE TITLE
Generated copyright header is missing a word

### ISSUE THREAD

**Issue by brjohnstmsft** [Azure/autorest#294]
The word "regenerated" is missing from the end of the header comment in generated C# files:
```
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License. See License.txt in the project root for
// license information.
// 
// Code generated by Microsoft (R) AutoRest Code Generator 0.11.0.0
// Changes may cause incorrect behavior and will be lost if the code is
```

Perhaps a bug in Template.WrapComment()?

**Comment by stankovski** [comment]
Fixed in #277 and included in 0.11.0

**Comment by brjohnstmsft** [comment]
Thanks!
